.github/actions/install-deps-and-canopy/action.yml

```diff
@@ -41,12 +36,8 @@ runs:
   - name: Install dependencies
     shell: bash
     if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
-    env:
-      CANOPY_EXTRAS: ${{ inputs.extras }}
-    run: poetry install --no-interaction --no-root --with dev --extras "${CANOPY_EXTRAS}"
+    run: make install POETRY_INSTALL_EXTRA_ARGS="--no-interaction --no-root --with dev"
   - name: Install project
     if: ${{ inputs.install-canopy == 'true' }}
     shell: bash
-    env:
-      CANOPY_EXTRAS: ${{ inputs.extras }}
-    run: make install-extras
+    run: make install POETRY_INSTALL_EXTRA_ARGS="--with dev"
```

Comment:
We should `--no-interaction` here as well

Reply:
Done